packages/syft/src/syft/core/tensor/smpc/context.py

```diff
@@ -0,0 +1,6 @@
+# stdlib
+# stdlib
+from typing import Optional
+
+GLOBAL_ARGS_ID: Optional[list] = None
```

Comment:
MAYBE: I am not sure we should keep the `args` and `kwargs` as a global context - maybe those require a better naming?
If those are exactly the `args`, `kwargs` that are received by a function in SMPC I don't think they should be kept as a global context because they should "make sense" only at the level of the function we want to run.

Reply:
I am feeling like we are moving away from the "Python language" (or any programming language) if we save the `args/kwargs` in the Global Action context and get them inside the action. I think the action should behave pretty much like a normal "function".

Regarding what should go in this Global Context - I think that the seeds/generators - everything SMPC related that requires context.

Reply:
Temporarily removed actions for the same.